app/src/main/java/com/x8bit/bitwarden/data/platform/manager/BridgeServiceProcessor.kt

```diff
@@ -0,0 +1,14 @@
+package com.x8bit.bitwarden.data.platform.manager
```

Comment:
Wanna rename the package to `processor` too

Reply:
Yep, good point, ty!! I moved the test too.

73df09e57e00fb94c88b4ca77e8d789a068ba490